### PR TITLE
Add automatic action for some MAD results

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-server/issues/14084

---

This patch adds new scanner flags that we'll use in the reviewer tools in a next iteration.

### CODE REVIEW

File: src/olympia/scanners/models.py [hunk 1]
```diff
@@ -387,3 +421,14 @@ class Meta(AbstractScannerResult.Meta):
 class ScannerQueryMatch(ModelBase):
     result = models.ForeignKey(ScannerQueryResult, on_delete=models.CASCADE)
     rule = models.ForeignKey(ScannerQueryRule, on_delete=models.CASCADE)
+
+
+class VersionScannerFlags(ModelBase):
+    version = models.OneToOneField(
+        'versions.Version', primary_key=True, on_delete=models.CASCADE
+    )
+    needs_human_review_by_mad = models.BooleanField(default=False)
```
Comment (diox) [inline src/olympia/scanners/models.py:430]: I don't have anything better to suggest yet, but I'm not a huge fan of this name.

Comment (willdurand) [reply src/olympia/scanners/models.py:430]: Yep. I was thinking that, given that we do not have a lot of scanners, we could have one field per scanner in the future (and stop relying on the version fields in some cases)

Comment (diox) [reply src/olympia/scanners/models.py:430]: Yeah, that's not my concern. It's mostly the `by_` that I don't like: It kinda implies the version needs a human review to be done by the mad scanner, at least that how I read it the first time and I was confused :)

Comment (willdurand) [reply src/olympia/scanners/models.py:430]: oh sorry, I misunderstood.

**Pull request merged**